Makefile

```diff
@@ -120,6 +120,10 @@ static: static-dependencies
 release-static: static
         # TODO(unassigned): cmake release flags + optimizations/hardening when we're out of alpha
 
+release-static-android: static-dependencies
+	mkdir -p $(build)
+	cd $(build) && $(cmake) $(cmake-debug) $(cmake-static) -D ANDROID=1 -D KOVRI_DATA_PATH="/data/local/tmp/.kovri" ../ && $(MAKE)
```

Comment:
i needed to copy the files. but i did it manually since i got permission denied for some symlinks in certificates. i still need to look at it